src/audio/sound_dose/Kconfig

```diff
@@ -0,0 +1,15 @@
+# SPDX-License-Identifier: BSD-3-Clause
+
+config COMP_SOUND_DOSE
+	tristate "Sound Dose module"
+	default m if LIBRARY_DEFAULT_MODULAR
+	depends on COMP_MODULE_ADAPTER
+        depends on IPC_MAJOR_4
```

Comment:
Is github web playing tricks or is the indentation off?

Reply:
My bad, it was spaces instead of tabs.